src/app/services/register/register-service.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { Registerdetails } from '../../models/register/register-details';
+// import { environment } from 'src/environments/environment.prod';
+
+const httpOptions = {
+  headers: new HttpHeaders({
+    'Content-Type': 'application/json',
+    'Access-Control-Allow-Headers': '*'
+  })
+};
+@Injectable({
+  providedIn: 'root'
+})
+export class RegisterServiceService {
+  // registerUrl = environment.authUrl;
+  registerUrl = 'http://127.0.0.1:8000/api/v1/auth';
```

Comment:
Hey @akram256, I recommend you use the remote Heroku API URL just so you actually tell that the feature will work on a live web server.